scipy/integrate/_bvp.py

```diff
@@ -712,7 +710,7 @@ def fun_jac_wrapped(x, y, p):
 
 
 def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
-              tol=1e-3, max_nodes=1000, verbose=0):
+              tol=1e-3, bc_tol=None, max_nodes=1000, verbose=0):
```

Comment:
The new keyword should be added as the last keyword, inserting it in the middle breaks backwards compatibility. (and I checked, `max_nodes` and `verbose` exist in scipy 1.2.0).

Reply:
That error has been corrected, thanks.